stdlib/3/sys.pyi

```diff
@@ -28,8 +28,8 @@ builtin_module_names: Sequence[str]  # actually a tuple of strings
 copyright: str
 # dllhandle = 0  # Windows only
 dont_write_bytecode: bool
-__displayhook__: Any  # contains the original value of displayhook
-__excepthook__: Any  # contains the original value of excepthook
+displayhook: Callable[[object], None]
+excepthook: Callable[[Type[BaseException], BaseException, TracebackType], None]
```

Comment:
Since the return value of these functions is (most likely, I did not check) ignored, the return type should be `Any`.